### PR TITLE
Run Travis against node 0.12 and the latest stable release

### PR DESCRIPTION
**Pull request by jeffposnick**
R: @addyosmani @gauntface @wibblymat 

Seems like a reasonable list of testing environments.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
   - "0.12"
-  - "0.11"
-  - "0.10"
+  - "5"
```
Comment (addyosmani) [inline .travis.yml]: 6.x.x is current (6.2.1). Any reason not to add this to the list too?

Comment (jeffposnick) [inline .travis.yml]: "5" is the latest major release on the list at https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Choosing-Node-versions-to-test-against

Comment (gauntface) [inline .travis.yml]: If you add 'stable' doesn't that run node 6?

Comment (addyosmani) [inline .travis.yml]: Interesting! ESLint added support for "6" recently so I was curious: https://github.com/eslint/eslint/commit/36da05264b94ae80d1cde172bae02b1f77808c4b

Comment (jeffposnick) [inline .travis.yml]: I see that `"node"` is an alias for the latest stable version, so I just switched to that.

Comment (addyosmani) [inline .travis.yml]: SGTM

**Comment by addyosmani** [comment]
LGTM

**Pull request merged**